airbyte-integrations/connectors/source-insightly/integration_tests/configured_catalog.json

```
@@ -0,0 +1,59 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "team_members",
+        "json_schema": {
+          "$schema": "http://json-schema.org/draft-07/schema#",
+          "type": "object",
+          "properties": {
+            "PERMISSION_ID": { "type": ["integer", "null"] },
+            "TEAM_ID": { "type": ["integer", "null"] },
+            "MEMBER_USER_ID": { "type": "integer" }
+          }
+        },
```

```suggestion
        "json_schema": {},
```
works and won't impact in the connector